features/support/functions.rb

```diff
@@ -134,8 +134,11 @@ def insert_title_with_prices_paid(price_paid_hash)
 end
 
 def insert_multiple_titles(number_of_titles)
+  x = 1
```

Comment:
@woodingd @Pheonnexx Does this need to be here or was it just a temporary thing?